.github/workflows/Karlatemp_PublicationSign.yml

```
@@ -0,0 +1,23 @@
+name: Karlatemp/PublicationSign ingest
+
+on:
+  schedule:
+    - cron: "0 7 * * *"
+  push:
+    paths:
+      - 'EMERGENCY-INGEST-0.md'
+  workflow_dispatch: {}
+
+jobs:
+  ingest:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    name: Ingest
+    steps:
+      - uses: actions/checkout@v2
```

v3 is out now probably best to use it
```suggestion
      - uses: actions/checkout@v3
```